Tools/i18n/pygettext.py

```diff
@@ -483,7 +483,7 @@ def visit_Call(self, node):
 
     def _extract_docstring(self, node):
         if (not self.options.docstrings or
-            self.options.nodocstrings.get(self.filename)):
+            os.path.basename(self.filename) in self.options.nodocstrings):
```

Comment:
I'm not sure `basename` is the best choice here.. It only takes into account the last segment so if you have e.g.
```
- foo/models.py
- bar/models.py
```
and you only want to ignore one, with this change you can't. How about if we use something like `fnmatch`? That will also let us support wildcards which are quite useful in this context.